build/NuSpecs/Microsoft.WindowsAppSDK.CompatibilitySetter.CS.targets

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(WindowsAppSDKRuntimePatchMode)' != '' or '$(WindowsAppSDKRuntimePatchMode2)' != '' or '$(WindowsAppSDKDisabledChanges)' != ''">
+      <WindowsAppSdkCompatibilityInitialize>true</WindowsAppSdkCompatibilityInitialize>
+  </PropertyGroup>
+
+  <Target Name="WinAppSDKCompatibilitySetterTarget" BeforeTargets="XamlPreCompile;CoreCompile"
```

Comment:
nit: we've tried to have the convention of WindowsAppSDKElementName for props, items, targets (though we haven't been consistent on SDK/Sdk case)